salt/modules/mysql.py

```diff
@@ -1612,12 +1598,33 @@ def _mysql_user_create(
                 else:
                     log.error("Auth via unix_socket can be set only for host=localhost")
```

Comment:
**question**: is this else necessary? Or, could it just be `elif host == "localhost"`? Also... what happens if unix_socket, but it's not `is_true`? Or is that even a possibility?

Reply:
**praise** Good catch.  That secondary check on whether unix_socket is True isn't necessary and was a hold over from a previous approach.